language/stdlib/modules/libra_account.move

```diff
@@ -388,9 +387,20 @@ module LibraAccount {
     // key `auth_key_prefix` | `fresh_address`
     // Creating an account at address 0x0 will cause runtime failure as it is a
     // reserved address for the MoveVM.
-    public fun create_unhosted_account<Token>(fresh_address: address, auth_key_prefix: vector<u8>)
+    // NOTE: This account can only be created in testnet
+    public fun create_testnet_account<Token>(fresh_address: address, auth_key_prefix: vector<u8>)
```

Comment:
So all testnet accounts are root VASPs with this credential?

Also, I think the comment above could use some updating.

Reply:
To you first question; yep! All accounts created (by default) are testnet accounts. One thing to note however, is that a RootVASP account published in this manner is _not_ the same as if it starts empty and then transitions to a RootVASP. In particular, a RootVASP account created in this manner will not be able to allow/have children. 

Updated the comment :)